packages/cosmic-swingset/s2.sh

```diff
@@ -9,3 +9,4 @@ EOF
   ag-cosmos-helper tx swingset deliver --keyring-backend=test $addr @/tmp/s2.json --gas=auto --gas-adjustment=1.05 --from=ag-solo -ojson --broadcast-mode=block --yes --output=json $opts
 fi
 ag-cosmos-helper query swingset mailbox $addr $opts
+#ag-cosmos-helper query swingset mailbox ${addr}z $opts
```

Comment:
did you commit that comment on purpose?

Reply:
Actually, the s2 and s2c shell scripts were just for reproducing a bug I was tracking down.

Not intended to be checked in.  I'll remove them from the repo.

Reply:
Of course, this raises the broader issue of how to write regression tests for cosmic-swingset.  We need a framework for this.

Reply:
If we "need" it in the sense that it's critical, then please open an issue for tracking.

Reply:
Done at #643